packages/textarea/src/LionTextarea.js

```diff
@@ -104,4 +105,15 @@ export class LionTextarea extends ObserverMixin(LionInput) {
   resizeTextarea() {
     autosize.update(this.inputElement);
   }
+
+  _slotPolyfillRendered() {
+    return new Promise(resolve => {
+      this.shadowRoot
+        .querySelector('slot[name="input"]')
+        .addEventListener('slotchange', async () => {
+          await this.updateComplete;
+          resolve();
```

Comment:
you said we need to wait for an `updateComplete` when the slot content has changed

I have difficulties to understand why, because it seems like if slot content has changed, then both the custom element + its shadow root are already rendered (otherwise there will be no slot and you count not listen) and the slot child is also rendered (because it is just a`<textarea>` added to it causing the `slotchange` event)

in other words, what is different before and after `updateComplete` which impacts the initialisation of autosize?

Reply:
As I've seen, when there is the need to polyfill the shadow dom and/or the slot, the light dom that composes the elements to be slotted on beginning to be part of the dom on the next `slotchanged` after the `updateComplete`, probably duo the lack of a proper shadow dom.
So the difference between before and after the `updateComplete` is that there is no slotted element rendered, especially the `setTextareaMaxHeight` need it to have it rendered with all its dimensions to calculate the `max-height`;

Reply:
Here are the print screens to show this behavior on Egde.

![Anotação 2019-07-27 093753](https://user-images.githubusercontent.com/13382278/61994588-c1fe8300-b052-11e9-90bb-9cdc47b4d217.png)

![Anotação 2019-07-27 093907](https://user-images.githubusercontent.com/13382278/61994589-c1fe8300-b052-11e9-8609-5206132f0ef1.png)

![Anotação 2019-07-27 094005](https://user-images.githubusercontent.com/13382278/61994590-c1fe8300-b052-11e9-8389-67a2d30d120d.png)

![Anotação 2019-07-27 094023](https://user-images.githubusercontent.com/13382278/61994591-c2971980-b052-11e9-9a25-0ed8bbafc14e.png)

![Anotação 2019-07-27 094047](https://user-images.githubusercontent.com/13382278/61994592-c2971980-b052-11e9-89e6-07b7e3473f6e.png)

![Anotação 2019-07-27 094119](https://user-images.githubusercontent.com/13382278/61994593-c2971980-b052-11e9-8dd9-62a828cb2782.png)

Reply:
Just in case you want to see that it was already rendered on the `firstUpdated` here is the first two screens shots on Opera.

![Anotação 2019-07-27 095017](https://user-images.githubusercontent.com/13382278/61994701-50273900-b054-11e9-8e21-2a48ead5322a.png)

![Anotação 2019-07-27 095124](https://user-images.githubusercontent.com/13382278/61994702-50273900-b054-11e9-9ffb-6f834d3ca102.png)